### ISSUE TITLE
"No such file or directory" error with new version 4.1.2

### ISSUE THREAD

**Issue by dwalintukan** [ConsenSys-archive/ganache-cli-archive#388]
I noticed this when my Travis CI build failed, but all the tests were passing on my machine.

**My Machine**
EthereumJS TestRPC v4.1.1 (ganache-core: 1.1.2)

**Travis CI Build**
ethereumjs-testrpc@4.1.2 

I am running the following script to allow for high value accounts:
```
#!/bin/bash
echo "Running TestRPC with high value accounts."

testrpc --account="0x5898b961541be6fed668c1fc08fd524f05e32c04da867a0c8c0c47d92e57f68a, 500000000000000000000000" --account="0x33fd84d1177554ae9c24812ca12c3ac9d870440335f1278c346bc9ec8376aa3f, 500000000000000000000000" --account="0x9915d43ed50e0f8cc7a4d6226366081cbcaaa3588312bc3714b0a178e521b4f8, 500000000000000000000000" --account="0x5ac9bdf80e118c22a0e5164d68143c463f18c14ce53fdaab9e9de48737304bec, 500000000000000000000000"
```

**Travis CI Logs**
```
$ nvm install 7
Downloading and installing node v7.10.1...
Downloading https://nodejs.org/dist/v7.10.1/node-v7.10.1-linux-x64.tar.xz...
######################################################################## 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v7.10.1 (npm v4.2.0)
install.yarn
$ node --version
v7.10.1
$ npm --version
4.2.0
$ nvm --version
0.33.4
install.1
20.10s$ npm install
9.18s$ npm install -g truffle
6.23s$ ./testrpc_high_value.sh & truffle test
Running TestRPC with high value accounts.
: No such file or directory
Could not connect to your Ethereum client. Please check that your Ethereum client:
    - is running
    - is accepting RPC connections (i.e., "--rpc" option is used in geth)
    - is accessible over the network
    - is properly configured in your Truffle configuration file (truffle.js)
```

The only difference was the TestRPC versions and when I forced an earlier version in Travis CI with `npm install -g ethereumjs-testrpc@4.1.1`, the tests passed and no more error.

## Expected Behavior

Running testrpc with custom accounts should not fail with error `no such file or directory`. Not sure if running normal `testrpc` command will cause the same error. 

## Current Behavior

Running testrpc with custom accounts fails with error `no such file or directory`.

## Steps to Reproduce (for bugs)

1. Install latest TestRPC 4.1.2
2. Install truffle
3. Run 
```
testrpc --account="0x5898b961541be6fed668c1fc08fd524f05e32c04da867a0c8c0c47d92e57f68a, 500000000000000000000000" --account="0x33fd84d1177554ae9c24812ca12c3ac9d870440335f1278c346bc9ec8376aa3f, 500000000000000000000000" --account="0x9915d43ed50e0f8cc7a4d6226366081cbcaaa3588312bc3714b0a178e521b4f8, 500000000000000000000000" --account="0x5ac9bdf80e118c22a0e5164d68143c463f18c14ce53fdaab9e9de48737304bec, 500000000000000000000000" & truffle test
```

## Context

Tests are not passing on Travis CI.

## Your Environment

* Version used: TestRPC 4.1.2
* Operating System and version: Ubuntu 14.04.5 LTS (on Travis CI)

**Comment by barchiesi** [comment]
The problems seems to be an extra `^M` at the end of the first line of `build/cli.node.js`.
Running the webpack build again fixes the problem.

**Comment by chfast** [comment]
Is there any workaround for this?

**Comment by smoove** [comment]
Can confirm, removing the `^M` in `build/cli.node.js` fixes the issue.

**Comment by noman-land** [comment]
Looks like this is related to non-Unix line endings in that file, which is breaking for Mac users.

**Comment by pablanopete** [comment]
Seconded that. removing the ^M worked for me too.

**Comment by tcoulter** [comment]
Hi all. This has been fixed in the latest release, v4.1.3. We're currently exploring how the ^M characters made their way in there. Thanks for your reports. 

Please reopen this issue if it is not fixed on your end. Cheers.
